io.openems.edge.goodwe.et/src/io/openems/edge/goodwe/et/ess/Config.java

```diff
@@ -19,9 +19,15 @@
 	@AttributeDefinition(name = "Is enabled?", description = "Is this Component enabled?")
 	boolean enabled() default true;
 
+	@AttributeDefinition(name = "Read-Only mode", description = "Enables Read-Only mode")
+	boolean readOnlyMode() default false;
+
 	@AttributeDefinition(name = "Modbus Unit-id", description = "Unit-id")
 	int unit_id() default GoodWeEtConstants.DEFAULT_UNIT_ID;
 
+	@AttributeDefinition(name = "Power Mode", description = "Mode of the power")
+	PowerModeEms Powermode() default PowerModeEms.AUTO;
```

Comment:
@venu-sagar I see that this config setting is not used anywhere. Also from the description it is not clear what it is supposed to be. Can we remove it?